release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -742,12 +749,12 @@ In the following tables, features are marked with the following statuses:
 |`compute.platform.openstack.rootVolume.type` for {rh-openstack}
 |Deprecated
 |Deprecated
-|Deprecated
+|Removed
```

Comment:
Confirm removal vs. continuing deprecation

Reply:
@pierreprinetti Would you be able to comment on this and the other deprecated item change?

Reply:
We haven't removed it, so still Deprecated.